db/models/patchArticleById.model.js

```diff
@@ -0,0 +1,38 @@
+const db = require("../connection");
+
+exports.editArticleById = (newVotes, id) => {
+  return checkArticleId(id).then((article) => {
+    if (!article) {
+      return Promise.reject({
+        status: 404,
+        msg: "that ID does not exist",
+      });
+    }
+
+    if (typeof newVotes !== "number") {
+      return Promise.reject({
+        status: 400,
+        msg: "Bad request",
+      });
+    }
```

Comment:
This should be handled by a PSQL error handler instead, because when it gets to the db query PSQL will fire off an error if the votes arent a number, so we should take advantage of this instead of using a custom err handler